projects/plugins/jetpack/modules/google-analytics/classes/wp-google-analytics-options.php

```diff
@@ -1,70 +1,109 @@
-<?php
-
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileNames
 /**
-* Jetpack_Google_Analytics_Options provides a single interface to module options
-*
-* @author allendav 
-*/
+ * Jetpack_Google_Analytics_Options provides a single interface to module options
+ *
+ * @author allendav
+ */
 
 /**
-* Bail if accessed directly
-*/
+ * Bail if accessed directly
+ */
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 
+/**
+ * Jetpack_Google_Analytics_Options main class.
+ */
 class Jetpack_Google_Analytics_Options {
+	/**
+	 * Get the Google Analytics tracking ID.
+	 *
+	 * @param string $option_name Nested 'jetpack_wga' option value to retrieve.
+	 * @param mixed  $default Default value if $option is not set.
+	 */
 	public static function get_option( $option_name, $default = false ) {
 		$o = get_option( 'jetpack_wga' );
 		return isset( $o[ $option_name ] ) ? $o[ $option_name ] : $default;
 	}
 
+	/**
+	 * Get the analytics tracking code.
+	 */
 	public static function get_tracking_code() {
 		return self::get_option( 'code', '' );
 	}
 
+	/**
+	 * Check if the tracking code is set.
+	 *
+	 * @return bool
+	 */
 	public static function has_tracking_code() {
 		$code = self::get_tracking_code();
 		return ! empty( $code );
 	}
 
-	// Options used by both legacy and universal analytics
+	/**
+	 * Get the 'anonymize_ip' option used by both legacy and universal analytics.
+	 */
```

Comment:
@anomiex Do you mean adding a cast to the return of `Jetpack_Google_Analytics_Options::get_option()` directly, or modifying each of the getter functions?

Reply:
Each of the getter functions. Changing `::get_option()` would break `::get_tracking_code()`.

Although now that I look closer, I think I was wrong about it possibly returning `''` or `'1'`. 😀 Core's `get_option()` has that behavior when you store/fetch a boolean directly, but when you store/fetch an array that contains booleans (as is what's happening here) they're preserved. I completely overlooked that it wasn't core's `get_option()` being used here.